x-pack/test/functional/apps/ml/anomaly_detection_jobs/index.ts

```diff
@@ -8,40 +8,55 @@
 import { FtrProviderContext } from '../../../ftr_provider_context';
 
 export default function ({ getService, loadTestFile }: FtrProviderContext) {
-  const esArchiver = getService('esArchiver');
+  const config = getService('config');
+  const security = getService('security');
+  const isCcs = config.get('esTestCluster.ccs');
+  const esNode = isCcs ? getService('remoteEsArchiver' as 'esArchiver') : getService('esArchiver');
   const ml = getService('ml');
 
   describe('machine learning - anomaly detection', function () {
     this.tags(['skipFirefox']);
 
     before(async () => {
-      await ml.securityCommon.createMlRoles();
-      await ml.securityCommon.createMlUsers();
+      if (isCcs) {
+        await security.testUser.setRoles(['superuser']);
+      } else {
+        await ml.securityCommon.createMlRoles();
+        await ml.securityCommon.createMlUsers();
+        await ml.securityUI.loginAsMlPowerUser();
+      }
     });
 
     after(async () => {
       // NOTE: Logout needs to happen before anything else to avoid flaky behavior
       await ml.securityUI.logout();
 
-      await ml.securityCommon.cleanMlUsers();
-      await ml.securityCommon.cleanMlRoles();
+      if (isCcs) {
+        await security.testUser.restoreDefaults();
+      } else {
+        await ml.securityCommon.cleanMlUsers();
+        await ml.securityCommon.cleanMlRoles();
+      }
 
-      await esArchiver.unload('x-pack/test/functional/es_archives/ml/farequote');
-      await esArchiver.unload('x-pack/test/functional/es_archives/ml/ecommerce');
-      await esArchiver.unload('x-pack/test/functional/es_archives/ml/categorization_small');
-      await esArchiver.unload('x-pack/test/functional/es_archives/ml/event_rate_nanos');
+      await esNode.unload('x-pack/test/functional/es_archives/ml/farequote');
```

Comment:
Just a nit, but esNode sounds nothing like esarchiver to me.  
Not a biggie but it threw me off.

Reply:
esNode has been what we call the es Archiver node regardless of whether or not it's local or remote. At this pint it is just the nomenclature we use in our CCS tests.